vita3k/gui/src/app_selector.cpp

```diff
@@ -31,7 +31,13 @@ using namespace std::string_literals;
 namespace gui {
 
 void init_user_apps(GuiState &gui, HostState &host) {
-    gui.apps_background.clear();
+    if (gui.live_area.settings) {
+        for (const auto &app : gui.apps_list_opened) {
+            if (app != "NPXS10015")
+                gui.apps_background.erase(app);
+        }
+    } else
```

Comment:
please add curly braces here, it makes it more readable especially since the if statement has braces

Reply:
Is not needed with one return value

Reply:
it works yes, but its a lot more readable to have braces

Reply:
no really more readable :) i no want touch it is fine here

Reply:
request approve, i no want change it here, is good like that

Reply:
review again, this code change is totaly removed, no need anymore ^^